src/worker.py

```diff
@@ -23,7 +23,6 @@
 DIMENSIONS_TO_MONO_RES = {
     (1280, 800): dai.MonoCameraProperties.SensorResolution.THE_800_P,
     (1280, 720): dai.MonoCameraProperties.SensorResolution.THE_720_P,
-    (640, 480): dai.MonoCameraProperties.SensorResolution.THE_480_P,
```

Comment:
This is resolution is actually unaccepted https://docs.luxonis.com/projects/hardware/en/latest/pages/articles/sensors/ov9282/#ov9282

Reply:
640x480 not supported!?